### DISCUSSION TITLE
Playing my own stream in the conference room

### DISCUSSION THREAD

**Discussion by dmitry027** [ant-media/Ant-Media-Server#5125] [Q&A]
Greetings everyone,

I have an issue with playing my own stream from remote server in the conference room. It can be reproduced using the demo conference file from AMS distribution (LiveApp/conference.html) with small modifications. I am doing the following:

1. Add the following line in the "publish_started" callback of conference.html file:
```
// Try to play my own stream from remote server once I started publishing it
webRTCAdaptor.play(obj.streamId, token, roomNameBox.value);

```
2. Open demo page and click "Join room" button to start publishing my stream to the room

What I expect:
I see additional video element with my stream from the remote server

What I get:
Alert with message "notSetRemoteDescription" and error message in the console:

```
webrtc_adaptor.js:908 set remote description is failed with error: InvalidAccessError: 
Failed to execute 'setRemoteDescription' on 'RTCPeerConnection': 
Failed to set remote offer sdp: The order of m-lines in subsequent offer doesn't match order from previous offer/answer.

(anonymous)	@	webrtc_adaptor.js:908
Promise.catch (async)		
takeConfiguration	@	webrtc_adaptor.js:906
wsConn.onmessage	@	websocket_adaptor.js:69
```
Ant Media version is 2.4.3

**Comment by tremzo** [comment]
Can you confirm why you want another player with your stream twice? It would mean every user in the conference will be seen twice? You can create a new manual stream alongside your conference and load it: `webRTCAdaptor.play("new-stream-id", token, roomNameBox.value);`

**Reply by dmitry027** [reply]
> Can you confirm why you want another player with your stream twice? It would mean every user in the conference will be seen twice? You can create a new manual stream alongside your conference and load it: `webRTCAdaptor.play("new-stream-id", token, roomNameBox.value);`

Thank you for the answer. My motivation is to let publisher see his/her own stream as it comes from the server (not directly from the webcam) for diagnostic purpose, so publisher can know how others see/hear him/her.
